README.md

```diff
@@ -286,6 +301,8 @@ installed regardless of your OS:
   2.8.12 or later.
```

Comment:
You should add a phrase similar to what you have with Bazel, "if using CMake for generating compilation targets."

Reply:
Done